bundler.d/test.rb

```diff
@@ -11,7 +11,7 @@
   gem 'launchy', '~> 2.4'
   gem 'facterdb', '~> 1.7'
   gem 'factory_bot_rails', '~> 5.0', :require => false
-  gem 'selenium-webdriver', :require => false
+  gem 'selenium-webdriver', '4.9.0', :require => false
```

Comment:
I'd separate this out.
```suggestion
  gem 'selenium-webdriver', :require => false
```